### DISCUSSION TITLE
Specific email notifications from Discussions

### DISCUSSION THREAD

**Discussion by aeslaughter** [community/community#59] [Discussions] [answered]
It would be useful if Github could distinguish notifications, such as using discussions@noreply.github.com

**Comment by rwcarlsen** [comment]
Yeah - this would really help with email filters and such to help better manage the sea of github notifications I get.

**Comment by becca** [comment, marked as answer]
Thank you for submitting this feedback! We don't have an easy way to distinguish the type of notification in email yet but if you wanted to try out our web notifications we have a way to create and save filters for Discussions (`is:discussions`)!

**Reply by aeslaughter** [reply]
Can you point me in the right spot for trying this out? I am looking at the notification settings in my account and don't see anything related to Discussions, am I in the right spot?

**Reply by becca** [reply]
Under https://github.com/settings/notifications you will want to be sure you have `web and mobile` selected (versus only `email`, for example). There's some documentation about this [here](https://docs.github.com/en/free-pro-team@latest/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#notification-delivery-options) and related documentation regarding filtering [here](https://docs.github.com/en/free-pro-team@latest/github/managing-subscriptions-and-notifications-on-github/about-notifications#customizing-notifications-and-subscriptions). The `is:discussions` filter isn't mentioned here yet but will be shortly!

**Reply by aeslaughter** [reply]
You mention "We don't have an easy way to distinguish the type of notification in email yet", does this imply that a method will be added to distinguish discussions from other notifications in email? If so, is there a timeline for that ability?

**Reply by becca** [reply]
I do not have a timeline for you at this time regarding this feature improvement.

**Reply by daviddavis** [reply]
We'd really love this feature on our project. It's impossible to sort email discussions notifications from other notifications like PR/issue comments.

**Reply by pmiossec** [reply]
@becca is it possible to have a way to differentiate discussions emails from issues emails?
Because at the moment, it's clearly a problem because in the email, we can't know that it is a discussion. 

Maybe adding 💬 in the title of the email?

**Comment by cbeams** [comment]
Glad to see this discussion is already underway. I tweeted about this problem earlier today at https://twitter.com/cbeams/status/1359081296270622721. Note in the second tweet the following theoretical, but not actually practical workaround:

> Technically, it is already possible to match on the `X-GitHub-Sender:` and `List-Post:` headers because their values include the string `repo-discussions`. However, that string comes after a newline and many clients' rules support (including Fastmail & Apple Mail) can't match on multiline values.

**Comment by jimmo** [comment]
We've just moved our project over to GitHub Discussions, and so far it's been a really fantastic experience. Thank you! However, we're seeing this problem too.

Because GMail filters can't do partial matches on the message-id, the best workaround I've come up with is

`listid:micropython.micropython.github.com -"micropython/micropython/repo-discussions"`
&nbsp; &nbsp; :arrow_right: Archive, Label: micropython
`listid:micropython.micropython.github.com "micropython/micropython/repo-discussions"`
&nbsp; &nbsp; :arrow_right: Archive, Label: micropython/discussions

which seems to work well enough. It would be really great if GitHub could put something into the emails to make this more obvious (a cc address would work well).
